debian/changelog

```diff
@@ -1,3 +1,12 @@
+e17 (0.21.3-3) experimental; urgency=medium
+
+  * Use build-time hardening
+  * Fixup build environment
+  * Use HTTPS for Vcs-* links
+  * debian/control: Use HTTPS for the website
+
+ -- Nicolas Braud-Santoni <nicolas@braud-santoni.eu>  Sun, 30 Oct 2016 02:37:30 +0200
+
```

Comment:
Could you regenerate without debian/changelog? gbp dch will generate it automatically at release to avoid merge conflicts along the way